js/application/init.js

```diff
@@ -242,7 +247,12 @@ define([
           document.title = this.config.title;
 
           // Launch the app specifics
-          this.main.start(this.config, dom.byId("viewDiv"), view, webscene, null, null);
+          this.main.start({
+            config: this.config,
+            viewDiv: dom.byId("viewDiv"),
+            view: view,
+            mapOrScene: webscene
```

Comment:
But they could also test `options.view instanceof MapView` and `options.mapOrScene instanceof WebMap` if their app happens to support both 2D and 3D.